app/tut/ga/cad-e-riomhchlaru.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Cad é Ríomhchlárú?
+prev: reamhra
+prev-text: Réamhrá Teagaisc
+---
+
+# Is é Ríomhchlárú ná...
+
+Sula dtosóimid ag foghlaim conas Setanta a scríobh agus a úsáid, ba chóir dúinn an ceist seo a chur:
+
+> Cad é ríomhchlárú?
+
+Inniu tá ríomhairí an chliste, is féidir leo áirimh a dhéanamh i bhfaiteadh na súl nach féidir le aon duine a dhéanamh ina cheann. Áfach, tá a lán cabhair ag teastáil ó ríomhairí agus is féidir linn iad a chabhrú le ríomhchlárú.
```

Comment:
(Níl fhios agam an é cabhair an focal ceart. Níl tú ag cabhrú leis an ríomhaire, tá tú ag tabhairt treoir dó)

Tá a lán cabhrach ag teastáil ó ríomhairí agus is féidir linn cabhrú leo tríd ríomhchlárú.

Mo mholadh:
Bíonn treoir de dhíth ar ríomhaire chun áireamh a dhéanamh agus is éard atá i ríomhchlárú ná an treoir sin a thabhairt.

Reply:
Chuir mé do mholadh isteach